bin/test-cleanup

```diff
@@ -9,7 +9,4 @@ fi
 
 echo "cleaning up namespace [${linkerd_namespace}] and associated test namespaces"
 
-kubectl delete ns "$linkerd_namespace"
-for ns in $(kubectl get ns | grep "^$linkerd_namespace-" | cut -f1 -d ' '); do
-  kubectl delete ns "$ns"
-done
+kubectl get ns | grep "^$linkerd_namespace-" | cut -f1 -d ' ' | xargs kubectl delete ns "$linkerd_namespace"
```

Comment:
> The script as written will return an error if I try to cleanup a namespace that doesn't exist, which then prompts me to cleanup the correct namespace.

I verified that the updated script will exit with a failure exit code (and prints a message) if there is no `linkerd` namespace and there are no `linkerd-*` namespaces. I think this is the right behavior; if there are `linkerd-*` namespaces then they should be deleted even if there is no `linkerd` namespace.

Reply:
> Also, I'm running kubectl v1.9.7 and I don't see the delay issue that you're seeing. Maybe it's a regression?

IDK. It might be a configuration setting on my part. However, the waiting behavior is very useful so I'd like to accomodate it.